source/concepts/environments/security.html.md.erb

```diff
@@ -39,7 +39,7 @@ We can see an overview of compliance across the Modernisation Platform and we wi
 
 ## Regional restrictions
 
-We restrict the regional usage of accounts that sit within the Modernisation Platform. We use a [Service Control Policy](https://github.com/ministryofjustice/aws-root-account/blob/main/terraform/organizations-service-control-policies.tf#L40) to do this.
+We restrict the regional usage of accounts that sit within the Modernisation Platform. We use a [Service Control Policy (SCP)](https://github.com/ministryofjustice/aws-root-account/blob/1ec842ec7b1356898bbca9cdb55f7dc64a9b6643/management-account/terraform/organizations-policy-service-control.tf#L40) to do this.
```

Comment:
As above, this should be `main` IMO to ensure we're not reviewing out-dated code.